solidity/contracts/BancorNetwork.sol

```diff
@@ -723,10 +656,10 @@ contract BancorNetwork is TokenHolder, ContractRegistryClient, ReentrancyGuard {
         address[] memory _path,
         uint256 _amount,
         uint256 _minReturn,
-        address _affiliateAccount,
-        uint256 _affiliateFee
+        address /* _affiliateAccount */,
+        uint256 /* _affiliateFee */
```

Comment:
@yudilevi do we really need to keep these interfaces?

Reply:
It is part of the API, and we don't want to break any application which relies on this.

Reply:
🦁 🦁 🦁